robot_skills/src/robot_skills/arms.py

```diff
@@ -194,6 +198,44 @@ def base_offset(self):
         """
         return self._arm.base_offset
 
+    def move_down_until_force_sensor_edge_up(self, timeout=10, retract_distance=0.01):
+        """
+        Move down the arm (hero specific, only joint arm_lift_joint) until the force sensor detects an edge up
```

Comment:
`robot_skills` are robot independent, so this funcionality belongs somewhere else.

Reply:
This is fixed for now using a subclass as Loy proposed. However this should be fixed differently, see #911